assets/scripts/app.js

```diff
@@ -20,3 +20,269 @@ const closeMenuHandler = () => {
+                            galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the
+                            printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                            ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s
+                            with the relea
+`,
+  },
+  {
+    id: 1,
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby', 'Css', 'JavaScript', 'Html'],
+    modalTagDesktop: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    modalTagMobile: ['Ruby on Rails', 'Css', 'Javascript'],
+    img: '../assets/images/modal-lg.png',
+    seeLive: 'http://ansderly.me/portfolio/',
+    seeSource: 'https://github.com/hansderly',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+                            industry. Lorem Ipsum has
+                            been
+                            the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the
+                            printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                            ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s
+                            with the relea
+`,
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby', 'Css', 'JavaScript', 'Html'],
+    modalTagDesktop: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    modalTagMobile: ['Ruby on Rails', 'Css', 'Javascript'],
+    img: '../assets/images/modal-lg.png',
+    seeLive: 'http://ansderly.me/portfolio/',
+    seeSource: 'https://github.com/hansderly',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+                            industry. Lorem Ipsum has
+                            been
+                            the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the
+                            printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                            ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s
+                            with the relea
+`,
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby', 'Css', 'JavaScript', 'Html'],
+    modalTagDesktop: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    modalTagMobile: ['Ruby on Rails', 'Css', 'Javascript'],
+    img: '../assets/images/modal-lg.png',
```

Comment:
- [ ] Please make sure the user is able to see the project image, your current path is corrupted as shown in this record below


![5](https://user-images.githubusercontent.com/77843050/208979970-9267b848-8b59-4add-933b-93a390b2c6f5.gif)

Reply:
A huge thanks to you @joskalenda , for taking the time to record and show me where I messed up.
thanks a lot.

Reply:
❤️ You are welcome @hansderly